OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -892,6 +892,32 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				if (engineVersion < 20150501)
+				{
+					// Change RenderFlare to RenderSprites + WithSpriteBody
+					var flares = node.Value.Nodes.Where(x => x.Key == "RenderFlare");
+					if (flares.Any())
```

Comment:
There is no point in having
`foreach(var flare in flares) flare.Key = "RenderSprites"`
because I still need to have
>if (flares.Any())
node.Value.Nodes.Add(new MiniYamlNode("WithSpriteBody", ...

outside the `foreach`.